### PR TITLE
Adding cytosim to conda-forge

### PR DESCRIPTION
**Pull request by SergeDmi**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by SergeDmi** [comment]
@conda-forge/help-python-c please review !

File: recipes/cytosim/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - python
+    - libblas
+    - liblapack
+    - pybind11
+
+test:
+  imports:
+    - cytosim
+about:
+  home: The package home page
```
Comment (ocefpaf) [inline recipes/cytosim/meta.yaml:41]: Please add a proper URL

```suggestion
  home: The package home page
```

File: recipes/cytosim/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  license: GPL-3.0-or-later
+  license_family: GPL3
+  license_file: LICENSE.txt
+  summary: "Cytosim: Langevin dynamics of active polymer networks"
+  doc_url: 
+  dev_url:
```
Comment (ocefpaf) [inline recipes/cytosim/meta.yaml:47]: Please remove it if you are going to leave them blank.

File: recipes/cytosim/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
+{% set name = "cytosim" %}
+{% set version = "0.0.18" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
```
Comment (ocefpaf) [inline recipes/cytosim/meta.yaml:9]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

File: recipes/cytosim/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
+{% set name = "cytosim" %}
+{% set version = "0.0.18" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
```
Comment (ocefpaf) [inline recipes/cytosim/meta.yaml:6]:
```suggestion
  name: {{ name|lower }}
  version: {{ version }}
```

File: recipes/cytosim/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 8234cbbd89fb897915abdabbefb80490e8044c92118363c002a1214ebd6fe01b
+build:
+  skip: true  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (ocefpaf) [inline recipes/cytosim/meta.yaml:14]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation 
  
```

**Review by ocefpaf — CHANGES_REQUESTED** [review]

**Comment by SergeDmi** [comment]
@ocefpaf  Solved I think !

File: recipes/cytosim/meta.yaml (hunk 1 above)
Comment (SergeDmi) [reply recipes/cytosim/meta.yaml]: Solved with package homepage

**Comment by SergeDmi** [comment]
@conda-forge/help-python-c please review !

File: recipes/cytosim/LICENSE.txt [hunk 6]
```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (ocefpaf) [inline recipes/cytosim/LICENSE.txt:1]: This file is in the tarball. No need to add it twice. Please remove it from the PR.

**Comment by SergeDmi** [comment]
Hey @ocefpaf , the linter requires a licence file, so I need to put it there !

File: recipes/cytosim/meta.yaml [hunk 7]
```diff
@@ -42,7 +42,6 @@ about:
   home: http://cytosim.org/
   license: GPL-3.0-or-later
   license_family: GPL3
-  license_file: LICENSE.txt
```
[the remark this answers is not in the document]
Comment (SergeDmi) [reply recipes/cytosim/meta.yaml]: Solved !

File: recipes/cytosim/LICENSE.txt (hunk 6 above)
Comment (SergeDmi) [reply recipes/cytosim/LICENSE.txt]: Actually it there is an issue when I run the docker build without a license file.
During the packaging of cytosim, I get : 

Fixing permissions
Traceback (most recent call last):
(...)
 raise ValueError(
ValueError: License file given in about/license_file (/home/conda/staged-recipes-copy/recipes/cytosim/LICENCE.txt) does not exist in source root dir or in recipe root dir (with meta.yaml)

##[error]Bash exited with code '1'.

File: recipes/cytosim/meta.yaml (hunk 7 above)
[the remark this answers is not in the document]
Comment (SergeDmi) [reply recipes/cytosim/meta.yaml]: Actually it there is an issue when I run the docker build without a license file.
During the packaging of cytosim, I get :

Fixing permissions
Traceback (most recent call last):
(...)
raise ValueError(
ValueError: License file given in about/license_file (/home/conda/staged-recipes-copy/recipes/cytosim/LICENCE.txt) does not exist in source root dir or in recipe root dir (with meta.yaml)

##[error]Bash exited with code '1'.

Any clue how I should proceed ?

File: recipes/cytosim/meta.yaml [hunk 8]
```diff
@@ -41,7 +41,8 @@ test:
 about:
   home: http://cytosim.org/
   license: GPL-3.0-or-later
-  license_family: GPL3
+  license_family: GPL3i
```
Comment (ocefpaf) [inline recipes/cytosim/meta.yaml]:
```suggestion
  license_family: GPL3
```

**Comment by SergeDmi** [comment]
Hey @ocefpaf , I am confused. 
As mentioned, if I don't put the licence file, the build fails : 

Fixing permissions
Traceback (most recent call last):
(...)
raise ValueError(
ValueError: License file given in about/license_file (/home/conda/staged-recipes-copy/recipes/cytosim/LICENCE.txt) does not exist in source root dir or in recipe root dir (with meta.yaml)

##[error]Bash exited with code '1'.


So should I put the licence file back in ?

File: recipes/cytosim/meta.yaml (hunk 8 above)
Comment (ocefpaf) [reply recipes/cytosim/meta.yaml]: Can you please show me where `GPL3i` is defined? As far as I know there is only `GPL3` family.

Comment (SergeDmi) [reply recipes/cytosim/meta.yaml]: I resolved it because it is fixed, it is back to GPL3.

Comment (ocefpaf) [reply recipes/cytosim/meta.yaml]: The browser showed it as outdated now. OK.

File: recipes/cytosim/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+    - pybind11
+
+test:
+  imports:
+    - cytosim
+about:
+  home: http://cytosim.org/
+  license: GPL-3.0-or-later
+  license_family: GPL3
+  license_file: LICENCE.txt
```
Comment (ocefpaf) [inline recipes/cytosim/meta.yaml:45]:
```suggestion
  license_file: LICENSE.txt
```

**Comment by ocefpaf** [comment]
> So should I put the licence file back in ?

Nope. Just use the correct file name as in the tarball. See my comment above.

**Comment by SergeDmi** [comment]
Fixed dumb spelling mistake.

File: recipes/cytosim/meta.yaml (hunk 7 above)
[the remark this answers is not in the document]
Comment (SergeDmi) [reply recipes/cytosim/meta.yaml]: Oupsies. Thanks for catching that.

**Pull request merged**